charts/project-origin-wallet/run_kind_test.sh

```diff
@@ -47,7 +48,41 @@ messageBroker:
   type: rabbitmqOperator
 EOF
 
+# setup operated
+cat << EOF > "$operated_filename"
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: wallet
+---
+apiVersion: rabbitmq.com/v1beta1
+kind: RabbitmqCluster
+metadata:
+  name: wallet-rabbitmq
+  namespace: wallet
+---
+apiVersion: postgresql.cnpg.io/v1
+kind: Cluster
+metadata:
+  name: cnpg-wallet-db
+  namespace: wallet
+spec:
+  instances: 3
+  storage:
+    size: 10Gi
+  bootstrap:
+    initdb:
+      database: wallet-database
+      owner: app
+  monitoring:
+    enablePodMonitor: true
+EOF
+
+kubectl apply -f "$operated_filename"
+
```

Comment:
RabbitMQ for wallet was moved to message-broker was it not? is this still relevant then?